drivers/sx126x/sx126x_netdev.c

```diff
@@ -288,9 +309,18 @@ static int _set_state(sx126x_t *dev, netopt_state_t state)
     case NETOPT_STATE_IDLE:
     case NETOPT_STATE_RX:
         DEBUG("[sx126x] netdev: set NETOPT_STATE_RX state\n");
+        /* Refer Section 4.2 RF Switch in Application Note (AN5406) */
+#if defined(BOARD_NUCLEO_WL55JC)
+        gpio_set(FE_CTRL1);
+        gpio_clear(FE_CTRL2);
+        gpio_set(FE_CTRL3);
```

Comment:
Isn't this only for debugging?

Reply:
Ah no sorry, got mixed up

Reply:
> should we maybe use a weak symbol here and implement board specific stuff in "board.h"? If more boards do the same, it will start growing like crazy

+1